gateway/test/src/test/resources/test-plan-data/policies/transfer-quota/009-probe-ip-transfer-quota.resttest

```diff
@@ -15,14 +15,13 @@ Content-Type: application/json
 {
   "TransferQuotaProbeResponse": {
     "config": {
-      "limit": 100,
+      "limit": 355,
       "granularity": "Ip",
-      "period": "Hour"
+      "period": "Day"
     },
     "status": {
-      "accepted": true,
-      "remaining": 100
```

Comment:
I've removed remaining because it seems to differ slightly by platform and we don't have a great way of putting in numeric ranges in currently with the RestTest suite. Maybe something for future.